packages/itwinui-react/src/core/Table/filters/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -114,26 +114,15 @@ export const DateRangeFilter = <T extends Record<string, unknown>>(
     });
   }, []);
 
-  const onKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
```

Comment:
Just out of curiosity, why did we remove the user's ability to submit filters by pressing Enter? Could we still allow `onKeyDown` and call `onSubmit`?

Reply:
We haven't removed any functionality. We are now using a `<form>` which automatically gets submitted when the user presses <kbd>Enter</kbd> from anywhere inside it.

Reply:
I see, that makes sense, `<form>` must do that automatically already.